frontend/src/js/controllers/authCtrl.js

```diff
@@ -142,8 +143,16 @@
                     onSuccess: function(response) {
                         if (response.status == 200) {
                             utilities.storeData('userKey', response.data.token);
+                            vm.redirectUrl = utilities.getData('redirectUrl');
```

Comment:
This change is for the user who isn't login and clicks `Host Challenge` on the landing page, then after login he will be redirected to `Create Challenge host Team page` directly. Please tell if I am not clear ?